### PR TITLE
psu: add imx6ull-ram.sdp script

### PR DESCRIPTION
**Pull request by damianloew**
JIRA: RTOS: 157

## Description

psu: add imx6ull-ram.sdp script

## Motivation and Context

Make running Phoenix-RTOS on imx6ull-evk possible and update quickstart for this platform. The current version of quickstart was adapted to feature/syspage with plo for imx6ull, which isn't available now.
 
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## How Has This Been Tested?

- [ ] Already covered by automatic testing.
- [ ] New test added: (add PR link here).
- [x] Tested by hand on: (imx6ull-evk).

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing linter checks and tests passed.
- [x] My changes generate no new compilation warnings for any of the targets.

## Special treatment

- [ ] This PR needs additional PRs to work (list the PRs, preferably in merge-order).
- [ ] I will merge this PR by myself when appropriate.

### CODE REVIEW

File: psu/imx6ull-ram.sdp [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+# Load the rest of kernel
+WAIT 0x15a2 0x80
+ERROR_STATUS
+
+# Write whole image
+WRITE_FILE F "phoenix-armv7a7-imx6ull.img" 0 0x80000000
+
+# Run kernel
+JUMP_ADDRESS 0x80000728
```
Comment (nalajcie) [inline psu/imx6ull-ram.sdp:18]: In all other scripts the jump adress is `0x80000718`. Are You sure it's correct?

Comment (damianloew) [reply psu/imx6ull-ram.sdp:18]: I saw that for both `imx6ull-flash.sdp` and `imx6ull-flash_fuses.sdp` JUMP_ADDRESS is `0x80000728` and I set the same value

Comment (HBuczynski) [reply psu/imx6ull-ram.sdp:18]: This address is ok. After changes introduced in **feature/syspage**, the plugin code has changed and `JUMP_ADDRES` is in the other place.

**Review by HBuczynski — APPROVED** [review]

File: psu/imx6ull-ram.sdp (hunk 1 above)
Comment (nalajcie) [reply psu/imx6ull-ram.sdp:18]: Ack. Please take care when merging into DTR project (update also SDP scripts in `_boot` dir.

**Pull request merged**